packages/tools/client-debugger/client-debugger-chrome-extension/src/devtools/DevtoolsScript.ts

```diff
@@ -22,17 +22,25 @@ import { formatDevtoolsScriptMessageForLogging } from "./Logging";
 
 console.log(formatDevtoolsScriptMessageForLogging("Initializing Devtools Script."));
 
-// When our extension view is launched, open the root visualization view.
-browser.devtools.panels.create(
-	"Fluid Framework Devtools",
-	"images/Icon.png",
-	"devtools/rootView.html",
-	(panel) => {
-		panel.onShown.addListener((window) => {
-			console.log(formatDevtoolsScriptMessageForLogging("Devtools view shown."));
-		});
-		panel.onHidden.addListener(() => {
-			console.log(formatDevtoolsScriptMessageForLogging("Devtools view hidden."));
-		});
-	},
-);
+// Default timeout for initialization is 30s.
+const initializationTimeout = 30_000;
+
+// Add a listener for when the button is clicked.
+browser.browserAction.onClicked.addListener(() => {
```

Comment:
Just wanted to clarify. This change makes it so that when we press the  button, we  open the root visualization view within 30s?

Also should it be browser_action?

Reply:
Right. Missing the content to be rendered initially. Clicking the icon is to re-render it. Making changes accordingly. Thx, Wayne. And in browser lib, it's browserAction